### DISCUSSION TITLE
convert with cmyk - color

### DISCUSSION THREAD

**Discussion by anmugerm** [ImageMagick/ImageMagick#2035] [Help]
I am trying to change the color of a generated QR code (png file). The QR code has only two colors (black and white) and was generated with phpqrcode.

White should become transparent and black should be colored to a desired color.

The conversion should work in both RGB and CMYK. In RGB it works with the following command:

convert test6066X.png -transparent white + opaque black -fill "rgb (0,255,0)" test6066.png

Now I want to do the same function with cmyk

convert test6066X.png -transparent white + opaque black -fill "cmyk (10,93,87,0)" -colorspace CMYK test6066.png

cmyk is apparently not a valid command, the message appears

/ usr / bin / convert convert: Unrecognized virtual pixel method
(cmyk (10.93.87.0)).

How can I convert to cmyk?

**Comment by snibgo** [comment]
What version of IM, on what platform?

What colorspace is test6066X.png in? sRGB, I suppose.

I don't understand your CMYK command. PNG files can't record CMYK values. So the input can't be CMYK, and the output can't be CMYK.
